contracts/connectors/loantoken/FactoryLoanToken.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity 0.5.17;
```

Comment:
so this supposed to be double proxy right?

Reply:
No, FactoryLoanToken is the proxy for factory loan tokens. It can be converted to the equivalence of LoanToken by setting the factory address to address(0). The double proxy for whitelisted tokens is with LoanTokenLogicResolver. All the iTokens will have that contract as their logic and then inside LoanTokenLogicResolver, the target will be set. This makes it a central point to change all the targets.